src/app/client/src/app/modules/shared/components/page-section/page-section.component.ts

```diff
@@ -159,4 +160,10 @@ export class PageSectionComponent implements OnInit {
       this.visits.emit(visits);
     }
   }
+  redirect() {
```

Comment:
emit event from pagesection and subscribe to it in the parent. on getting event redirect.